src/test/kotlin/com/pubnub/api/legacy/endpoints/pubsub/SubscribeEndpointTest.kt

```diff
@@ -557,4 +559,57 @@ class SubscribeEndpointTest : BaseTest() {
             request.queryParameter("state").firstValue()
         )
     }
+
+    @Test
+    fun subscribeCanReturnMessageTypeAndSpaceId() {
+        val expectedMessageType = MessageType("thisIsCustomMessageType")
+        val expectedSpaceId = SpaceId("thisIsSpaceId")
+        stubFor(
+            get(urlPathEqualTo("/v2/subscribe/mySubscribeKey/coolChannel/0"))
+                .willReturn(
+                    aResponse().withBody(
+                        """
+                            {
+                              "t": {
+                                "t": "14607577960932487",
+                                "r": 1
+                              },
+                              "m": [
+                                {
+                                  "a": "4",
+                                  "f": 0,
+                                  "i": "Client-g5d4g",
+                                  "p": {
+                                    "t": "14607577960925503",
+                                    "r": 1
+                                  },
+                                  "k": "someSubKey",
+                                  "c": "coolChannel",
+                                  "d": {
+                                    "text": "Enter Message Here"
+                                  },
+                                  "b": "coolChan-bnel",
+                                  "e": 0,
+                                  "mt": "${expectedMessageType.type}",
```

Comment:
Shouldn't it be "expectedMessageType.value" instead of "expectedMessageType.type"?